[docs] Prohibit opening front-end devtools (#5233)
DevExpress/testcafe: docs/articles/documentation/guides/basic-guides/run-tests.md

  * [Console Shortcuts in Live Mode](#console-shortcuts-in-live-mode)
* [Quarantine Mode](#quarantine-mode)

> > Important! Front-end development tools (such as React DevTools, or Vue DevTools) can interfere with TestCafe and cause errors. Do not open them when you are running or debugging TestCafe tests.

Reviewer: ```suggestion
> Important! Front-end development tools (such as React DevTools or Vue DevTools) can interfere with TestCafe and cause errors. Do not open them while you run or debug TestCafe tests.
```
